pkgs/applications/video/mpv/scripts/quality-menu.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, oscSupport ? false
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "mpv-quality-menu";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "christoph-heinrich";
+    repo = "mpv-quality-menu";
+    rev = "v${version}";
+    hash = "sha256-93WoTeX61xzbjx/tgBgUVuwyR9MkAUzCfVSrbAC7Ddc=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/mpv/scripts
+    cp quality-menu.lua $out/share/mpv/scripts
+  '' + lib.optionalString oscSupport ''
+    cp quality-menu-osc.lua $out/share/mpv/scripts
```

Comment:
Would be nice to have this automatically installed by the wrapper too. Perhaps a `passthru.extraScripts` argument?

Reply:
Yeah, having some sort of builder for mpv scripts could be nice. But I'm not gonna work on it because I don't even use any mpv scripts, I was just requested to package something.